src/index.d.ts

```diff
@@ -1487,6 +1573,20 @@ declare namespace admin.auth {
      */
     deleteUser(uid: string): Promise<void>;
 
+    /**
+     * Deletes the users specified by the given uids.
+     *
+     * Deletes multiple existing users. Deleting a non-existing user won't
```

Comment:
Please document array length constraints.

Reply:
Oops; I hadn't updated this based on the design doc. I've added the full docs now. (Which includes the 1000 thing, but also touches on the rate limitation.)